site/topics/memory/esap-ram.rst

```diff
@@ -16,6 +16,75 @@ Address Register and RAM
 RAM Module
 ==========
 
+* As previously discussed, memory stores data in individually indexable memory locations called addresses
+* The number of unique memory addresses is called the address space
+* Each location stores some amount of data, called the addressability
```

Comment:
I've never heard the term _addressability_ used in this context. Word.

Reply:
It does feel odd to use it _this way_. 

I feel like the word is used more to describe the resolution (for lack of a better term) of memory's ability to address data. Like, a measure of how fine we can get data out of RAM. In a typical comp, it's byte addressable, meaning we can pin point individual bytes, which is cool. 

However, the flip side is, that's how much data is stored in each address' location. For the purposes here, it's effectively talking about the same thing, _but_ the wording does feel odd.